pkg/kubelet/kubelet_test.go

```diff
@@ -377,7 +377,7 @@ func newTestKubeletWithImageList(
 
 	var prober volume.DynamicPluginProber // TODO (#51147) inject mock
 	kubelet.volumePluginMgr, err =
-		NewInitializedVolumePluginMgr(kubelet, kubelet.secretManager, kubelet.configMapManager, token.NewManager(kubelet.kubeClient), allPlugins, prober)
+		NewInitializedVolumePluginMgr(kubelet, kubelet.secretManager, kubelet.configMapManager, token.NewManager(kubelet.kubeClient), allPlugins, prober, nil)
```

Comment:
Prefer using `go.opentelemetry.io/otel/trace.NewNoopTracerProvider()` to nil in tests.

Reply:
Done